Bankomat/Models/CardTransaction.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bankomat.Models
+{
+    public class CardTransaction : CardsData
```

Comment:
Kodel sitoje klaseje nera jokiu duomenu apie tai is kokios ir i kuria saskaita buvo padaryta tranzakcija, kokia buvo suma?